v3/widgets/plot.go

```diff
@@ -202,12 +206,14 @@ func (self *Plot) Draw(buf *Buffer) {
 	self.Block.Draw(buf)
 
 	maxVal := self.MaxVal
+	minVal := self.MinVal
 	if maxVal == 0 {
 		maxVal, _ = GetMaxFloat64From2dSlice(self.Data)
+		minVal, _ = GetMinFloat64From2dSlice(self.Data)
```

Comment:
So if `maxVal == 0`, the `minVal` will be re-calculated even user sets this value by purpose?